assets/js/modules/pagespeed-insights/components/DashboardPageSpeed.js

```diff
@@ -77,61 +83,56 @@ export default function DashboardPageSpeed() {
 	}
 
 	const reportData = strategy === STRATEGY_MOBILE ? reportMobile : reportDesktop;
-	const footerLinkHTML = sprintf(
-		/* translators: 1: link attributes, 2: translated service name */
-		__( 'View details at <a %1$s>%2$s</a>', 'google-site-kit' ),
-		`href="${ addQueryArgs( 'https://developers.google.com/speed/pagespeed/insights/', { url } ) }" class="googlesitekit-cta-link googlesitekit-cta-link--external" target="_blank"`,
-		_x( 'PageSpeed Insights', 'Service name', 'google-site-kit' )
-	);
+
+	// Update the active tab for "In the Lab" or "In The Field".
+	const updateActiveTab = ( dataSrcIndex ) => {
+		if ( dataSrcIndex === 0 ) {
```

Comment:
@aaemnnosttv I think we'd have to wrap the `TabBar` component with another component in order to do this, or create another callback function which calls `updateActiveTab()`. Is it worth it to creating those to avoid using the index?